bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Remove all of this the model has already be loaded before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>